workflow/rules/common.smk

```diff
@@ -31,7 +31,7 @@ def read_samples():
 samples = read_samples()
 SAMPLES = list(sorted(set(samples["sample_name"])))
 GROUPS = list(sorted(set(samples["group"])))
-CATEGORIES = ["coding", "regulatory", "intronic"]
+CATEGORIES = ["coding", "regulatory", "intronic", "discarded"]
```

Comment:
The "discarded" group was deliberately left out to get smaller reports -- depending on the data, this may lead to way too many entries.

Reply:
Is this problematic, though? It is in a separate table, anyways, so users will only see those if they actively go looking. So unless this really blows up the report, I'd rather keep it than have a non-functional link (or special-case the `discarded` category).

Reply:
From experience, this can really blow up the report, but I too think that these should be included. I've renamed the category to "other" to reflect that it can contain circles which may be of interest, but simply aren't included in the other three categories.
Perhaps there could be a config param which allows fine tuning the categories or something like that. Or perhaps categories aren't the way to go, but rather have one large table that can be filtered and searched properly (so basically a datavzrd problem with respect to very large tables), with accompanying stats.

Reply:
Or filtering via varlociraptor probabilities? Or are these all circles that already pass the FDR?

Reply:
Yes, the variants are already those surviving the fdr control step

Reply:
Hmm, OK. But then they should definitely be in there. So let's keep it to your rename for now.